docs/cli.rst

```diff
@@ -1,11 +1,271 @@
+
+
+Running Tasks and Flows
+***********************
+When you know the specific task or flow you want to run, execute it with the ``run`` command.
+
+.. code-block:: console
+
+    $ cci task run <name> --org <org> [options]
+    $ cci flow run <name> --org <org> 
+
+This runs the respective task or flow ``<name>`` against the org ``<org>``. (You can see a list of available orgs by running ``cci org list``.)
+
+    Example: The ``run_tests`` task executes Apex unit tests. If you have an org called ``dev``, you can run this task against it with the command ``cci task run run_tests --org dev``.
+
+Tasks usually require additional options to be passed when using the ``cci task run`` command.
+
+
+Task Info & Options
+*******************
+
+For additional information on task ``<name>``:
+
+    $ cci task info <name>
+
+Information about specific tasks includes:
+
+* A description of what the task does.
+* The particular Python class associated with this task.
+* The syntax for running the command.
+* Any options for the task.
+
+Information about specific task options includes:
+
+* The syntax for the option (``-o <name> value``).
+* If the option is required or optional.
+* A description of what the option does.
+
+An example of a task's information and options:
+
+.. code-block:: console
+
+    $ cci task info util_sleep
+    util_sleep
+
+    Description: Sleeps for N seconds
+
+    Class: cumulusci.tasks.util.Sleep
+
+    Command Syntax
+
+        $ cci task run util_sleep
+
+    Options
+
+        --seconds SECONDS
+        Required
+        The number of seconds to sleep
+        Default: 5
+
+For additional information on flow ``<name>``:
+
+    $ cci flow info <name>
+
+Information about specific flows includes:
+
+* A description of the flow.
+* The ordered steps (and substeps) of a flow.
+
+An example of a flow's information and options:
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+        when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
```

Comment:
It might be helpful to add explanation here.

```suggestion
The flow information for ``dev_org`` shows that it's composed of three subflows, ``dependencies``, ``deploy_unmanaged``, and ``config_dev``, and one task, ``snapshot_changes``. The tasks and flows making up the three subflows are also listed.
```

Reply:
That's helpful indeed. I'd even move it up above the code sample and combine it with the lead-in to the sample. "In this example of a flow's information and options, the flow information for ``dev_org``...